app/src/main/java/com/android/shelfLife/model/foodItem/FoodItemRepository.kt

```diff
@@ -47,4 +50,8 @@ interface FoodItemRepository {
   fun selectFoodItem(foodItem: FoodItem?)
 
   fun setFoodItems(householdId: String, value: List<FoodItem>)
+
+  fun setIsGenerated(value: Boolean)
```

Comment:
This should not be called IsGenerated. We are not generating a fooditem, we are quick adding it